iot/iot-hub/quickstart.mdx

```diff
@@ -36,7 +32,7 @@ In this Quickstart, we show you how to create your first IoT Hub to connect obje
 
     - **Shared plan:** This is a cost-effective, straightforward plan. Good for proofs-of-concept or DIY projects. The price is kept low by sharing resources between users, but it is therefore forbidden to use MQTT features which require memory (no messages with `QoS1`, `QoS2`, `retained` and no anonymous devices). Although this plan does not limit the number of devices, you will be billed for devices exceeding the number allowed within the free tier.
     - **Dedicated Plan:** This plan offers dedicated resources for the Hub, which means more predictable performances and full support of the MQTT protocol.
-    - **HA Plan:** Not only are resources dedicated to the Hub, but they are also replicated so that the broker is highly available. This setting is ideal for workloads that require greater robustness for their Hub.
+    - **High Availability (HA) Plan:** Not only are resources dedicated to the Hub, but they are also replicated so that the broker is highly available. This setting is ideal for workloads that require greater robustness for their Hub.
```

Comment:
```suggestion
    - **High Availability (HA) Plan:** Not only are resources dedicated to the Hub, they are also replicated so that the broker is highly available. This setting is ideal for workloads that require greater robustness for their Hub.
```